### DISCUSSION TITLE
Public wiki, Submit pages for Review

### DISCUSSION THREAD

**Discussion by jarrodlovas** [requarks/wiki#5252] [Help / Questions]
Is there a method that would allow users who wish to contribute to a public wiki to be able to create pages but cannot publish or would the best approach be to do the following:

1. Create a group that is added to those that register
2. Setup Rules for pages to only be created in / Review for example and that directory is hidden from everyone
3. Then I can review and move it into the appropriate directory once reviewed?

What are people's thoughts? or what have others done for public community ran wikis

**Comment by ayoahha** [comment]
Hi there
I am looking for the same behaviour... Can we achieve a sort of moderation before put article or modifications of articles online ?

I was wondering if creating a directory "/drafts" could do the job, with special group and permissions, but i guess there will be some issues 
- with already existing articles, merging ...
- multiple modifications of an article
What do you think ?

**Comment by fnafox** [comment]
You can do this by setting up an external git repository where people can suggest changes and enable external edit.
`Settings > General > Edit Shortcuts > Display External Edit Button`
If you set this up with two way git synchronization you can approve it on git and have the wiki pull it automatically.
